packages/jest-util/Readme.md

```diff
@@ -0,0 +1,66 @@
+# [@jest/utils](https://github.com/jestjs/jest/blob/v29.6.4/packages/jest-util/src/index.ts)
+
+This packages is a collection of utilities and helper functions
+
+## [clearLine.ts](https://github.com/jestjs/jest/blob/v29.6.4/packages/jest-util/src/clearLine.ts)
+- It ensures that the clearing operation is only performed when running in a terminal environment, and not when the output is redirected to a file or another non-terminal destination.
```

Comment:
```suggestion
It ensures that the clearing operation is only performed when running in a terminal environment, and not when the output is redirected to a file or another non-terminal destination.
```

I don't think we need this as a bullet point?

Reply:
Sure will commit the suggested changes soon 🚀